Fixes #10138 - Expand information about variable attributes (#10200)
MicrosoftDocs/PowerShell-Docs: reference/5.1/Microsoft.PowerShell.Core/About/about_Classes.md

Notice **Slots** property isn't shown in `$r1` output. However, the size was
changed by the constructor.

## Static keyword

The `static` keyword defines a property or a method that exists in the class
and needs no instance.

A static property is always available, independent of class instantiation. A
static property is shared across all instances of the class. A static method is
available always. All static properties live for the entire session span.

> ### Example using static keywords and methods

Reviewer: I think this is more accurately about static _properties_ and methods, using the static keyword.

```suggestion
### Example using static properties and methods
```

----- -----

Exception setting "Brand": "The argument is null or empty. Provide an
argument that isn't null or empty, and then try the command again."
At C:\tmp\Untitled-5.ps1:11 char:1
+ $dev.Brand = ""
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting
```

> For a complete list of available attributes, see
> [about_Functions_Advanced_Parameters][03].

Reviewer: Nit: Not _technically_ a complete list - just the list of built-in attributes that are applied to parameters.

Maybe instead:

```suggestion
For more information on available attributes, see
[about_Functions_Advanced_Parameters][03].
```
